run_server.sh

```diff
@@ -17,10 +17,11 @@
 TEAM_ID="$1"
 TEAM_SECRET="$2"
 PORT="$3"
-RELAY_ADDRESS="$4"
+PERSISTENT="$4"
+RELAY_ADDRESS="$5"
 
-if [[ "TEAM_ID" == "" || "$TEAM_SECRET" == "" || "$PORT" == "" ]] ; then
-  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT> [RELAY ADDRESS]'
+if [[ "TEAM_ID" == "" || "$TEAM_SECRET" == "" || "$PORT" == "" || "$PERSISTENT" == "" ]] ; then
+  echo 'usage: <TEAM ID> <TEAM SECRET> <PORT> <PERSISENT> [RELAY ADDRESS]'
```

Comment:
persistent is misspelled in line 24.
Either add a single line for each parameter that explains what the parameter is used for, or add a line that points to the README (and add the usage information to the README).